app/shared/recurring-reservation-controls/RecurringReservationControls.js

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from 'react';
+import ControlLabel from 'react-bootstrap/lib/ControlLabel';
+import FormControl from 'react-bootstrap/lib/FormControl';
+import FormGroup from 'react-bootstrap/lib/FormGroup';
+import Select from 'react-select';
+
+import { injectT } from 'i18n';
+
+function createOptionRenderer(t) {
+  return option => t(option.label);
+}
+
+function createValueRenderer(t) {
+  return option => t(option.label);
```

Comment:
Actually the `option.label` is correct there as well. The valueRenderer tells how to display the selected value in the input. But good that you noticed.